apps/surfer/src/locations/Dialog.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRef } from 'react';
+import { DialogAppSDK } from '@contentful/app-sdk';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { KeyValueMap } from 'contentful-management';
+import { useSurfer } from '../hooks/useSurfer';
+import { SurferRpcCommands } from '../types';
+import { SurferContainer } from '../components/SurferContainer';
+
+export type DialogParameters = {
+  shareToken: string;
+};
+
+const Dialog = () => {
+  const { parameters, close } = useSDK<DialogAppSDK<KeyValueMap, DialogParameters>>();
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const { isLoading } = useSurfer(containerRef, 'draft_configuration', {
+    shareToken: parameters.invocation.shareToken,
+    onReady: ({ requestView, configureView }) => {
+      configureView({
+        disableDraftConfiguration: true,
+        configurationOnCancelOverride: true,
+      });
+      requestView('draft_configuration');
+    },
+    onRpcMessage({ command }) {
```

Comment:
`onRpcMessage: ({ command }) => {}` or `onReady() {}` to have a consisten style

Reply:
✅